Cargo.toml

```diff
@@ -126,7 +128,7 @@ embassy-executor = { git = "https://github.com/kaspar030/embassy", branch = "for
 embassy-hal-internal = { git = "https://github.com/kaspar030/embassy", branch = "for-riot-rs-2024-09-19" }
 embassy-nrf = { git = "https://github.com/kaspar030/embassy", branch = "for-riot-rs-2024-09-19" }
 embassy-net = { git = "https://github.com/kaspar030/embassy", branch = "for-riot-rs-2024-09-19" }
-embassy-rp = { git = "https://github.com/kaspar030/embassy", branch = "for-riot-rs-2024-09-19" }
+embassy-rp = { git = "https://github.com/elenaf9/embassy", branch = "for-riot-rs-2024-09-19_multicore" }
```

Comment:
So this contains a patch disabling some interrupts.  why is that necessary? can we make that a feature in `embassy-rp` and PR it to our fork at kaspar030/embassy?

Reply:
now I spotted the [explanation](https://github.com/future-proof-iot/RIOT-rs/pull/397#discussion_r1745532593).

Reply:
see https://github.com/future-proof-iot/RIOT-rs/pull/397#discussion_r1808728314